tests/ballerina-tools-integration-test/src/test/resources/packaging/Package.md

```diff
@@ -1,37 +0,0 @@
-# Test Connector
```

Comment:
Why is this removed?

Reply:
Previously before pushing we manually created the package zip to be pushed. But now we have used the init command to create the project and the build command to build the package zip. So we no longer need this.